### ISSUE TITLE
Better README File.

### ISSUE THREAD

**Issue by starteNCS** [lasernoises/egrikor#1]
You should add a better README File.

Maybe add what egrikor is, some visual samples and a quick start guide?

**Comment by lasernoises** [comment]
You are absolutely right. I'm working on one now. I don't think anything like a quickstart guide makes sense at this stage of the project, but I can add a minimal example and some screenshots.
